Public/Src/Cache/ContentStore/Library/Tracing/EvictResult.cs

```diff
@@ -116,11 +116,17 @@ public DeleteResult ToDeleteResult(ContentHash contentHash)
             return new DeleteResult(DeleteResult.ResultCode.ContentNotDeleted, contentHash, EvictedSize, PinnedSize);
         }
 
+        /// <nodoc />
+        public TimeSpan Age => CreationTime - LastAccessTime;
+
+        /// <nodoc />
+        public TimeSpan? EffectiveAge => CreationTime - EffectiveLastAccessTime;
+
         /// <inheritdoc />
         public override string ToString()
         {
             return Succeeded
-                ? $"Success Size={EvictedSize} Files={EvictedFiles} Pinned={PinnedSize} LastAccessTime={LastAccessTime} Age={CreationTime - LastAccessTime} ReplicaCount={ReplicaCount} EffectiveLastAccessTime={EffectiveLastAccessTime} EffectiveAge={CreationTime - EffectiveLastAccessTime}"
+                ? $"Success Size={EvictedSize} Files={EvictedFiles} Pinned={PinnedSize} LastAccessTime={LastAccessTime} Age={Age} ReplicaCount={ReplicaCount} EffectiveLastAccessTime={EffectiveLastAccessTime} EffectiveAge={EffectiveAge}"

```

Comment:
>LastAccessTime={LastAccessTime} [](start = 88, length = 31)

Weren't we going to remove LastAccessTime and EffectiveLastAccessTime, in favor of just having Age and EffectiveAge?

Reply:
We trace the eviction manually instead of using `Result.ToString`.